auto_submit/lib/requests/github_webhook.dart

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:logging/logging.dart';
+import 'package:shelf/shelf.dart';
+
+final Logger logger = Logger('GithubWebhook');
+
+class GithubWebhook {
```

Comment:
dartdoc?

Reply:
No, dartdoc refers to adding a class doc here like

```dart
/// Handler for processing GitHub webhooks.
///
/// On events where a label was added to a pull request, ...
```

See https://dart.dev/guides/language/effective-dart/documentation for the general style guide and suggestions on writing effective documentation

Reply:
Gotcha, will add it now. Thanks

Reply:
FYI this would be above line 10 on the `class GithubWebhook`